doc/api.apib

```diff
@@ -174,11 +183,342 @@ aglio -i doc/api.apib -o api.html --theme-template triple --theme-variables stre
 + Request (application/json)
 
         {
-            "docker_image": "busybox:latest"
+            "docker_image": "luigi-canary-dummy:dummy",
+            "commit_sha": "f043333085fa87738ac24f04d64fb58ecc845111",
+        }
+
++ Response 200 (application/json)
+
+        {
+            "enabled": null ,
+            "docker_image": "luigi-canary-dummy:dummy",
+            "commit_sha": "f043333085fa87738ac24f04d64fb58ecc845111",
+            "next_natural_trigger": null
+        }
+
+# Group Resources
+
+## Resources [/v2/resources]
```

Comment:
Lacks documentation somewhere. It is non-obvious how the ids can be invented, and what concurrency means here. It's also too abstractly documented (no example) [in Confluence](https://confluence.spotify.net/display/DEX/Styx#Styx-yaml).